tool/common/deps.bzl

```diff
@@ -8,7 +8,9 @@ def vaticle_dependencies_ci_pip():
 
```

Comment:
Added TOML library to handle `Cargo.toml`, same one we use in `bazel-distribution` and elsewhere.